lock-api/src/main/java/com/palantir/lock/watch/NoOpLockWatchValueCache.java

```diff
@@ -34,4 +34,7 @@ public void ensureStateRemoved(long startTimestamp) {}
 
     @Override
     public void onSuccessfulCommit(long startTimestamp) {}
+
+    @Override
+    public void dumpState() {}
```

Comment:
Do we want to also log "no op" here, like in the other cases?

Reply:
yes can do